File: Day1ProgrammingFoundation/day1_7.java
import java.util.Scanner;

public class day1_7{
    public static void main(String[] args) {

        final int MIN_AGE = 21;
        final double MIN_SALARY = 3000000;

        Scanner sc = new Scanner(System.in);

        System.out.println("Masukkan nama customer: ");
        String name = sc.nextLine();

        System.out.println("Masukkan usia customer: ");
        int age = Integer.parseInt(sc.nextLine());

        System.out.println("Masukkan penghasilan customer: ");
        double salary = Double.parseDouble(sc.nextLine());

        if (age < MIN_AGE){
            System.out.println(name + " tidak eligible secara usia");
        } else if (salary < MIN_SALARY){
            System.out.println(name + " tidak eligible secara penghasilan");
        } else if (age >= MIN_AGE && salary >= MIN_SALARY){
            System.out.println(name + " eligible");
        } else{
            System.out.println(name + " tidak eligible");
        }


    }
}

File: Day1ProgrammingFoundation/day1-7.py
MIN_AGE = 21
MIN_SALARY = 3000000

namaCustomer = input("Masukkan nama anda: ")
umur = int(input("Masukkan umur anda: "))
penghasilan = int(input("Masukkan penghasilan anda: "))

if umur < MIN_AGE:
    print(namaCustomer, "Tidak eligible secara usia")
elif penghasilan < MIN_SALARY:
    print(namaCustomer, "Tidak eligible secara penghasilan")
elif umur >= MIN_AGE and penghasilan >= MIN_SALARY:
    print(namaCustomer, "Konsumen Eligible")
else:
    print("Konsumen tidak eligible secara umur dan penghasilan")




